### PR TITLE
quarter stream ID has a legal range, state it and punishment for brea…

### LINKED ISSUE #93: Quarter Stream ID range is smaller than Stream ID range
Stating the obvious here. But we're using value with a legal range larger than semantically legal. RFC 9000 has some similar caveats with MAX_STREAMS https://www.rfc-editor.org/rfc/rfc9000.html#section-19.11

To address this, the suggestion is to add something like

>  The largest legal stream ID is 2^62 - 1, so this value cannot exceed (2^62-1) / 4. Receipt of a frame that includes a larger value MUST be treated as a connection error of type FRAME_ENCODING_ERROR

### PR DESCRIPTION
**Pull request by LPardue**
…king hte law

Also, switch to format to beatuiful superscript HTML rendering as used by RFC 9000.

fixes #93

### CODE REVIEW

File: draft-ietf-masque-h3-datagram.md [hunk 1]
```diff
@@ -161,10 +161,13 @@ HTTP/3 Datagram {
 Quarter Stream ID:
 
 : A variable-length integer that contains the value of the client-initiated
-bidirectional stream that this datagram is associated with, divided by four.
-(The division by four stems from the fact that HTTP requests are sent on
+bidirectional stream that this datagram is associated with, divided by four (The
```
Comment (DavidSchinazi) [inline draft-ietf-masque-h3-datagram.md:164]: nit: if you remove the dot, then remove the following uppercase?
```suggestion
bidirectional stream that this datagram is associated with, divided by four (the
```

File: draft-ietf-masque-h3-datagram.md [hunk 2]
```diff
@@ -161,10 +161,13 @@ HTTP/3 Datagram {
...
+division by four stems from the fact that HTTP requests are sent on
 client-initiated bidirectional streams, and those have stream IDs that are
-divisible by four.)
+divisible by four.). The largest legal QUIC stream ID value is 2<sup>62</sup>-1,
+so the largest legal value value of Quarter Stream ID is 2<sup>62</sup>-1 / 4.
```
Comment (DavidSchinazi) [inline draft-ietf-masque-h3-datagram.md:168]: That's just too much value :P
```suggestion
so the largest legal value of Quarter Stream ID is 2<sup>62</sup>-1 / 4.
```

**Review by DavidSchinazi — APPROVED** [review]
Nice. Approved modulo a couple nits

File: draft-ietf-masque-h3-datagram.md (hunk 1 above)
Comment (LPardue) [reply draft-ietf-masque-h3-datagram.md:164]: derp

File: draft-ietf-masque-h3-datagram.md [hunk 3]
```diff
@@ -161,10 +161,13 @@ HTTP/3 Datagram {
 Quarter Stream ID:
 
 : A variable-length integer that contains the value of the client-initiated
-bidirectional stream that this datagram is associated with, divided by four.
-(The division by four stems from the fact that HTTP requests are sent on
+bidirectional stream that this datagram is associated with, divided by four (the
+division by four stems from the fact that HTTP requests are sent on
 client-initiated bidirectional streams, and those have stream IDs that are
-divisible by four.)
+divisible by four.). The largest legal QUIC stream ID value is 2<sup>62</sup>-1,
```
Comment (DavidSchinazi) [inline draft-ietf-masque-h3-datagram.md:167]: Well now we have an extra dot I think
```suggestion
divisible by four). The largest legal QUIC stream ID value is 2<sup>62</sup>-1,
```

**Pull request merged**